### PR TITLE
Verb change

### PR DESCRIPTION
**Pull request by astrozzc**
Update config of cost-managment permission
    
    The new cost-managment permissions are staying in CI. So
    separate the update in a new commit.

Change structure of permission config to support description
    
    Change the list of verbs from string to json so support adding description to
    the permission.
    
    JIRA: https://issues.redhat.com/browse/RHCLOUD-8979

### CODE REVIEW

**Review by wcmitchell — APPROVED** [review]
👍

File: configs/permissions/cost-management.json [hunk 1]
```diff
@@ -1,10 +1,35 @@
...
+    ],
+    "openshift.project": [
+        {"verb":"*"},
+        {"verb":"read"}
+    ],
+    "cost_model": [
+        {"verb":"*"},
+        {"verb":"read"},
+        {"verb":"write"},
+        {"verb":"read"}
```
Comment (coderbydesign) [inline configs/permissions/cost-management.json:30]: Looks like a duplicate here?

Comment (astrozzc) [reply configs/permissions/cost-management.json]: Good catch! Updated

**Review by coderbydesign — COMMENTED** [review]
I checked these out locally and things seed as expected. One nit might be to format the JSON files? It might help with linting diffs moving forward.

**Review by coderbydesign — APPROVED** [review]

**Pull request merged**